__tests__/components/editor/ResourceTemplate.test.js

```diff
@@ -37,7 +38,17 @@ describe('<ResourceTemplate />', () => {
   })
 
   it('displays Alert', () => {
+    expect(wrapper.find(Alerts).length).toEqual(1)
+  })
+
+  describe('When there is an error and no resource template', () => {
     const alertWrapper = shallow(<ResourceTemplate.WrappedComponent errors={['Oooops']} />)
-    expect(alertWrapper.find(Alerts).length).toEqual(1)
+    it('displays Alert', () => {
+      expect(alertWrapper.find(Alerts).length).toEqual(1)
+    })
```

Comment:
This looks like the same stuff on lines 17 + 40-42. Intended to test it in two places, or cruft?

Reply:
Different tests. In the first case, there is a resource template; in the second there is not. The code handles differently.